themes/src/main/resources/theme/keycloak.v2/account/src/app/content/linked-accounts-page/LinkedAccountsPage.tsx

```diff
@@ -132,15 +126,15 @@ class LinkedAccountsPage extends React.Component<LinkedAccountsPageProps, Linked
                 <PageSection isFilled variant={PageSectionVariants.light}>
                     <Stack hasGutter>
                         <StackItem>
-                            <Title headingLevel="h2" size='2xl'>
+                            <Title headingLevel="h2" className="pf-u-mb-lg" size='xl'>
                                 <Msg msgKey='linkedLoginProviders'/>
                             </Title>
                             <DataList id="linked-idps" aria-label='foo'>
                                 {this.makeRows(this.state.linkedAccounts, true)}
                             </DataList>
                         </StackItem>
                         <StackItem>
-                            <Title headingLevel="h2" size='2xl'>
+                            <Title headingLevel="h2" className="pf-u-mt-xl pf-u-mb-lg" size='xl'>
```

Comment:
Try adding a <TextContent> around the <Title> and then remove the utility classes.